index.js

```diff
@@ -14,9 +14,12 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+var log4js = require('ep_etherpad-lite/node_modules/log4js')
+var statsLogger = log4js.getLogger("stats");
```

Comment:
cool I didn't know this existed :P

Reply:
It didn't. From other cases it seemed like I could put an arbitrary string in there for the logger name, and it worked when I tried it. If there's an existing logger that seems appropriate I could switch it.

Reply:
hah awesome, no that's cool and makes total sense :)  I hate extending log4js though, I wanna rip it out at some point or something.  We're using an old version because they changed their API under us ;(